test/backwards_compatibility_test.go

```
@@ -0,0 +1,256 @@
+	}
+	fmt.Println("Environment output:", environmentOutput)
+
+	// Extract environment ID
+	environmentID := extractID(environmentOutput)
+	if environmentID == "" {
+		t.Fatal("Failed to extract environment ID")
+	}
+
+	// Clean up
+	fmt.Println("\nCleaning up...")
+	runCommand("delete", "environment", "--environment", environmentID)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)